### PR TITLE
Fix ViTForMaskedImageModeling example in documentation

### PR DESCRIPTION
**Pull request by fxmarty**
Following https://github.com/huggingface/transformers/pull/22152, `logits` is not the right output key name.

cc @alaradirik @amyeroberts

### CODE REVIEW

**Comment by ydshieh** [comment]
@fxmarty I am just going to open a PR, but you are too fast! Thanks.

**Comment by fxmarty** [comment]
@ydshieh It could be I missed it elsewhere, so feel free to push here / do an other PR!

**Comment by ydshieh** [comment]
You have to check your CircleCI setting however. The CI has issue to run in this PR.

**Comment by ydshieh** [comment]
OK, nothing missed, but as the CI has some problem, I am going to open PR.

**Comment by ydshieh** [comment]
I will make sure you are a contributor in #22186 @fxmarty . Close this one as mentioned above.

**Comment by fxmarty** [comment]
Ah yes I did not configure SSO with CircleCI, maybe that's the issue.
